### PR TITLE
Pythonanywhere free-tier compatibility

### PR DESCRIPTION
**Pull request by adelra**
This branch can be used in PythonAnywhere service because it doesn't use fake_ua library (which uses some external database of useragents which is blocked in PythonAnywhere).

### CODE REVIEW

**Comment by dimisdas** [comment]
Hello,

fake_ua supports defaulting to a user agent if it fails, I think. It would be best to use that, combined with your random array.

`ua = fake_useragent.UserAgent(fallback='Your fallback ua string')`

and perhaps even wrap it around a try:except.

**Comment by adelra** [comment]
Please check the latest commit. I couldn't find the fake_ua's offline mode, there is only one method that uses cache, but in the case of PythonAnywhere as that is the first time running fake_ua, it wouldn't work.

**Comment by dimisdas** [comment]
Looks good, but the changed formatting messes up with my whole local tree big time. Too many conflicts. Could you pass me write access (found on the sidebar of your pull request)? Thank you!

File: src/instabot.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
-from fake_useragent import UserAgent
+import random
```
Comment (dimisdas) [inline src/instabot.py]: already imported, todo: remove

**Comment by dimisdas** [comment]
Works on my end. Can you verify it's also working on pythonanywhere?

**Comment by adelra** [comment]
Thanks for the editing, just tried it on PythonAnywhere and it works like a charm you may merge this PR now.

**Pull request merged**